tools/depends/target/python3/Makefile

```diff
@@ -10,28 +10,43 @@ ARCHIVE=$(SOURCE).tar.xz
 BASE_URL=https://www.python.org/ftp/python/$(VERSION)
 
 ifeq ($(findstring apple-darwin, $(HOST)), apple-darwin)
-  HOSTPLATFORM=_PYTHON_HOST_PLATFORM="darwin"
-  LINK_ICONV=-framework CoreFoundation -liconv
+  LDFLAGS+= -framework Foundation
+  ifeq ($(OS), darwin_embedded)
+    EXTRA_CONFIGURE=ac_cv_func_wait3=no ac_cv_func_wait4=no ac_cv_func_waitpid=no \
+                    ac_cv_func_execv=no ac_cv_func_fexecv=no ac_cv_func_getentropy=no \
+                    ac_cv_func_setpriority=no ac_cv_func_sendfile=no ac_cv_header_sched_h=no \
+                    ac_cv_func_posix_spawn=no ac_cv_func_posix_spawnp=no \
+                    ac_cv_func_forkpty=no ac_cv_lib_util_forkpty=no \
+                    ac_cv_func_getgroups=no
+  endif
 endif
 
-ifeq ($(OS), darwin_embedded)
-  EXTRA_CONFIGURE=ac_cv_func_wait3=no ac_cv_func_wait4=no ac_cv_func_waitpid=no \
-                  ac_cv_func_execv=no ac_cv_func_fexecv=no ac_cv_func_getentropy=no \
-                  ac_cv_func_setpriority=no ac_cv_func_sendfile=no ac_cv_header_sched_h=no \
-                  ac_cv_func_posix_spawn=no ac_cv_func_posix_spawnp=no \
-                  ac_cv_func_forkpty=no ac_cv_lib_util_forkpty=no \
-                  ac_cv_func_getgroups=no
+ifeq ($(CROSS_COMPILING),yes)
+	CROSSBUILD=--host=$(HOST) \
+			   --build=$(BUILD)
 endif
 
-# configuration settings
-CONFIGURE=./configure --prefix=$(PREFIX) \
-  --disable-shared \
-  --without-ensurepip \
-  --disable-framework --with-system-expat --with-system-ffi --without-pymalloc --enable-ipv6 \
-  $(EXTRA_CONFIGURE)
+COMPILE_HOST_PATH := $(PATH)
 
+export PATH=$(NATIVEPREFIX)/bin:$(COMPILE_HOST_PATH)
+export LINK_ICONV:=$(LINK_ICONV)
 
-LIBDYLIB=$(PLATFORM)/libpython3.8.a
```

Comment:
I'm not sure if you meant to remove this or not but we typically use this to tell if a package has been compiled or not. This is beneficial when rebuilding depends and the package doesn't have to be rebuilt for no reason.

Reply:
Was a reason, I think it was the file never exists in that location.

Reply:
Going with the original, python still repeatedly builds. I was incorrect about libpython3.8.a being available, but for some reason i cant get it to acknowledge it exists and therefore to not do the make/make install.

Removing the libdylid step, putting the make/make install in the .installed-$(platform) step correctly recognises a successful build/rebuild.

So after all that, short answer is this works, the previous stuff does not.